etc/docker/dev/docker-compose.yml

```diff
@@ -26,6 +29,8 @@ services:
       - "5432:5432"
     command: ["-c", "fsync=off","-c", "synchronous_commit=off","-c", "full_page_writes=off"]
   graphite:
+    hostname: graphite
+    container_name: graphite
```

Comment:
please restore this file to the original :)

Reply:
:smile: